devfile.yaml

```diff
@@ -1,17 +1,19 @@
 schemaVersion: 2.1.0
 metadata:
   name: microprofile-quickstart
-attributes:
-  che-theia.eclipse.org/sidecar-policy: mergeImage
 components:
   - name: tools
+    container:
+      image: registry.redhat.io/devspaces/udi-rhel8:3.0
+      memoryLimit: 3Gi
+      volumeMounts:
+        - name: m2
+          path: /home/user/.m2
+        - name: gradle
+          path: /home/user/.gradle
+  - name: eap
```

Comment:
not sure if it matters but this is the eap-xp image, which is different from a pure eap image

Reply:
renamed